CIP-0008/README.md

```diff
@@ -388,26 +289,153 @@ We recommend usage of unprotected headers vs protected headers when possible. Th
+4) Is simple to implement
+
+In particular, using fnv32a over CRC32 frees up 1024 bytes of memory due to not having a lookup table which is significant on hardware wallets.
+
+#### Blake2b [[RFC 7693](https://tools.ietf.org/html/rfc7693)]
+
+Blake2b is a hash algorithm used commonly in Cardano. Notably, Blake2b-224, Blake2b-256 and Blake2b512 are used depending on the context.
+
+#### base64url [[RFC 4648 section 5](https://tools.ietf.org/html/rfc4648#section-5)]
+
+`base64url` allows encoding bytes in a human-readable format that is also safe to pass in URLs.
+
+#### Other blockchain standards
+
+Other blockchains have existing specifications for message signing, but they mostly revolve around scripts trying to validate messages. We don't leverage any of their work in particular but it may be of interest.
+
+- [BIP-137](https://github.com/bitcoin/bips/blob/master/bip-0137.mediawiki) - simply scheme for message signing that works with P2PKH, P2PSH and bech32
+- [BIP-322](https://github.com/kallewoof/bips/blob/master/bip-0322.mediawiki) - reuses Bitcoin script to process a generic signed message format
+
+- [EIP-191](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md) - encode data for Ethereum contracts
+- [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) - encode structs for Ethereum contracts
+
+### Existing Code
+
+#### Cryptography
+
+Cardano already allows message signing within the WASM bindings. Notably,
+
+1) [sign](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L279)
+2) [verify](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L322)
+
+You can see an example of these two functions [here](https://repl.it/repls/FlusteredSimpleFunction)
+
+Even if you use cryptographically secure `sign` and `verify` functions, you still have the following problems:
+
+1) No human-recognizable prefix
+0) No error detection
+0) User could accidentally sign a transaction or a block thinking it's harmless data
+
+We also have a risk of a few different kinds of replay attacks
+
+4) A dapp asks person A to sign "BOB" and then another dapp asks user B to sign "BOB". B can just use the signature from A
+0) A dapp asks person A to sign "BOB" on a testnet chain. Person B then sends this signed message to the same dapp running on mainnet (same argument applies to sidechains)
+
+### Reference implementations
+
+- [COSE-JS](https://github.com/erdtman/cose-js)
+- [Rust message signing](https://github.com/Emurgo/message-signing)
+
+### Unresolved
+
+This specification provides no means of Revocation.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [x] There are wallets supporting creation of signed messages as per this protocol (enumerated 2023-12-19 as per [CardanoBallot list of wallets](https://voting.summit.cardano.org/user-guide) supporting CIP-8 signed messages):
```

Comment:
add some of these to `Implementors` in the header?

Reply:
I wouldn't jump to do that because it might become a case of wallet chauvinism over a standard that is pretty universally supported by wallets now... and I believe the main purpose of this line is to indicate that _receiving_ these messages is supported by a well known, widely used dApp.

Reply:
Make sense

And also wallets implement the CIP30 version of CIP05 (which has some limitations) so may not even count as `Implementors` at the header level.